tests/kernel/install_ltp.pm

```diff
@@ -400,16 +405,20 @@ sub run {
 
     log_versions 1;
 
-    zypper_call('in efivar') if is_sle('12+') || is_opensuse;
+    if (is_alp) {
+        assert_script_run("transactional-update -n -c pkg install efivar", 90);
+    } else {
+        zypper_call('in efivar') if is_sle('12+') || is_opensuse;
+    }
 
     $grub_param .= ' console=hvc0' if (get_var('ARCH') eq 'ppc64le');
     $grub_param .= ' console=ttysclp0' if (get_var('ARCH') eq 's390x');
     if (!is_sle('<12') && defined $grub_param) {
         add_grub_cmdline_settings($grub_param, update_grub => 1);
     }
 
-    add_custom_grub_entries if (is_sle('12+') || is_opensuse) && !is_jeos;
-    setup_network;
+    add_custom_grub_entries if (is_sle('12+') || is_opensuse || is_transactional) && !is_jeos;
+    setup_network unless is_transactional;
```

Comment:
LGTM. The only question is why not run network setup for transactional. Would it be too complicated or testing not relevant? Anyway, we need to remember not run network tests for transactional.

Reply:
I should mentioned it into PR, it is more complicated at the moment for September prototype which will just cover cve and syscalls. There will be follow up to extend testing.